### PR TITLE
Final touch

### PR DESCRIPTION
**Pull request by cynthiainga**
- [ ] Add the maxlength and minlength to the input form
- [ ] Change the animation for the refresh button, it animates just as we refresh the page.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -33,12 +33,16 @@ <h2>Add your score</h2>
...
+                minlength="4"
+                maxlength="15"
                 required
               />
               <input
                 type="number"
                 class="input-score"
                 placeholder="Your Score"
+                min="0"
+                max="9000000000"
```
Comment (Nemwel-Boniface) [inline src/index.html:45]: ## Hello Cynthia :wave: 
## :recycle:  [OPTIONAL CHANGES]  :recycle: 
you did a great job with adding some form validations. 
I however feel that a value of ```9000000000``` is too large and un reasonable  for a competition. I would recommend a value like 1000.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @cynthiainga  :wave: :wave:  

Your project is complete!  
Great job implementing the form validations. It now feels more natural and very professional :sunglasses: :sunglasses: 

I have left you one optional change  that I feel is worth the change

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Even the miyagi-do master approves :partying_face: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (cynthiainga) [reply src/index.html:45]: Yeah, I know the number is to much. It was just to test but I will modify the number

Comment (Nemwel-Boniface) [reply src/index.html:45]: Okay that's great

**Pull request merged**